docs/installation.md

```diff
@@ -0,0 +1,45 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Installing Federation v2](#installing-federation-v2)
+  - [Prerequisites](#prerequisites)
+    - [Creating Clusters](#creating-clusters)
+    - [Deployment Image](#deployment-image)
+  - [Helm Chart Deployment](#helm-chart-deployment)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Installing Federation v2
+
+## Prerequisites
+
+Federation v2 requires Kubernetes v1.11 or newer.
+
+### Creating Clusters
+
+The following is a list of Kubernetes environments that have been tested and are supported by the Federation v2 community.
+
+- [kind](./environments/kind.md)
+- [Minikube](./environments/minikube.md)
+- [Google Kubernetes Engine (GKE)](./environments/gke.md)
+- [IBM Cloud Private](./environments/icp.md)
+
+After completing the steps in one of the above guides, return here to continue the deployment.
+
+**IMPORTANT:** You must set the correct context in your cluster(s) using the command below.
+
+```bash
+kubectl config use-context cluster1
+```
+### Deployment Image
+
+If you follow this user guide without any changes you will be using the latest
```

Comment:
I think the helm chart actually defaults to `canary` rather than `latest`.  #701 is intended to ensure a released chart that sets the appropriate image tag rather than requiring it to be manually set, but in advance of that it might make sense to mention specifying a released tag (e.g. `v0.0.7`).